apps/req.c

```diff
@@ -490,7 +490,6 @@ int req_main(int argc, char **argv)
             if (oid_bio == NULL) {
```

Comment:
Why try to proceed if this fails?  The user specified an OID file and you're not opening it.  Should use ```bio_open_default```, right?

Reply:
> Why try to proceed if this fails? The user specified an OID file and you're not opening it. Should use `bio_open_default`, right?

I don't know either. 
We might ask whoever did that.